finder-app/writer.c

```diff
@@ -0,0 +1,74 @@
+/***********************************************************************
+* Filename : writer.c
+* Author : Maanas Makam Dileep Kumar
+* Description : C file to write a given string of text into a file.
+* Date : 01/29/2023
+* References : Man pages and Week2 lectures
+************************************************************************/
+#include <stdio.h>
+#include <stdlib.h>
+#include <syslog.h>
+#include <fcntl.h>
+#include <string.h>
+#include <errno.h>
+#include <unistd.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+
+int main(int argc, char *argv[])
+{
+	// check number of arguments
+	if( argc != 3)
+	{
+		printf("Error : Invalid number of arguments\n\r");
+		printf("Usage : ./writer <filename> <string>\n\r");
+		return 1;
+	}
+
+	// extract the filepath and string
+	char *filename = argv[1];
+	char *text = argv[2];
```

Comment:
Nice idea to save as variable names.  What do we know about the access of these variables regarding write and how could we use the compiler to help us enforce?